live/app/templates/assessments/get-comparison.hbs

```diff
@@ -0,0 +1,5 @@
+{{comparison-window
+    answer=model.answer
+    challenge=model.challenge
```

Comment:
Est-ce que ce ne serait pas intéressant de passer par les [positionals-params](https://guides.emberjs.com/v2.11.0/components/passing-properties-to-a-component/#toc_positional-params) ?